sample/src/main/java/com/canhub/cropper/sample/SMainActivity.kt

```diff
@@ -42,13 +48,26 @@ internal class SMainActivity : AppCompatActivity() {
             hideButtons(binding)
             supportFragmentManager
                 .beginTransaction()
+                .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN)
                 .replace(R.id.container, SCameraFragmentJava.newInstance())
                 .commit()
         }
+
+        onBackPressedDispatcher.addCallback {
+            if (binding.container.visibility == View.VISIBLE) {
```

Comment:
maybe I'm missing something, when do we set `binding.container.visibility = GONE` ?

Reply:
It is set as `android:visibility="gone"` in the xml file

Reply:
indeed, but when it became gone again? like when we return to this from another screen we would need to set Gone again, right? Sorry if I'm missing something obvious.